data/themes/darktable.css

```diff
@@ -1948,6 +1949,15 @@ spinbutton>button
 {
   border-top: 2.5px solid @border_color;
```

Comment:
With triangle, border top is maybe too big. I suggest to try with 1.5px (or 2px max) only. That's actually a little too big with the triangle now.

Reply:
you're right of course... done !